opm/input/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -325,6 +356,31 @@ namespace Opm {
 
    };
 
+    class EclipseGridLGR: public EclipseGrid
+    // Specialized Class to describe LGR refined cells. 
+    {
+    public:
+      using vec_size_t = std::vector<std::size_t>;
+      EclipseGridLGR() = default;
+      EclipseGridLGR(const std::string& self_label, const std::string& father_label_, 
+                     int father_lgr_level, size_t nx, size_t ny, size_t nz, 
+                     vec_size_t father_lgr_index);
+      ~EclipseGridLGR() = default;
+      vec_size_t getFatherGlobalID() const;
+      void set_lgr_global_counter(std::size_t counter){
```

Comment:
Just curious—was there a particular reason for naming it set_lgr_global_counter() rather than setLgrGlobalCounter()? I noticed a similar pattern with get_father_global(), save_all_lgr_labels(), and others. I was wondering about the choice of naming style.